api/app/src/main/kotlin/packit/controllers/IndexController.kt

```diff
@@ -0,0 +1,19 @@
+package packit.controllers
+
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import packit.model.Packet
+import packit.service.IndexService
+
+@RestController
+@RequestMapping("/index")
```

Comment:
Sorry I didn't catch this before. I think a more natural naming convention here would have `PacketController` with` @RequestMapping("/packet")` and then the below the method is `fun index()` with `@GetMapping("/")`. Then later we will also have a `fun packet(id)` with `@GetMapping("/<id>")`